source/includes/oai-pmh/_root.md

```diff
@@ -32,8 +59,14 @@ a ``429 Too Many Requests`` HTTP error if you exceed the limit.
 
 ## Metadata formats
 
-Metadata for each record is available in several formats. The available formats
-include:
+```python
```

Comment:
Again I think it would be nice to see the [protocol](http://www.openarchives.org/OAI/openarchivesprotocol.html#ListMetadataFormats) mentioned  or partially explained in the text segment. Maybe add something like: "you can list the following formats via:" since we are then analyzing them underneath.